src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseHandler.cs

```diff
@@ -71,6 +76,12 @@ static DatabaseHandler()
             {
                 recoveryModelEnums.Add(displayRecoveryModels[key], key);
             }
+
+            // Azure SLO info is invariant of server information, so set up static objects we can return later
+            AzureEditions = GetAzureEditions();
+            AzureBackupLevels = GetAzureBackupLevels();
+            AzureMaxSizes = GetAzureMaxSizes(AzureEditions);
```

Comment:
Instead of passing in the display names, it seems better to have GetAzureEditions actually return the AzureEditions and passing that around instead.